pkg/utils/metrics/metrics.go

```diff
@@ -0,0 +1,102 @@
+/*
+metrics tracks request behavior inside tenant (which manages Ensign user accounts,
+projects, and topics) and quarterdeck (which manages authentication and authorization)
+*/
+package metrics
+
+import (
+	"fmt"
+	"sync"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/rs/zerolog/log"
+)
+
+// Prometheus namespaces for the collectors defined in this package.
+const (
+	NamespaceHTTPMetrics = "http_stats"
+)
+
+var (
+	// Active HTTP requests
+	ActiveTenantRequests prometheus.Gauge
+	ActiveQDRequests     prometheus.Gauge
```

Comment:
I think this is a very interesting idea and I'd be interested to see how it worked. Most request _should_ take < 500ms and the scrape interval is 1 - 15s, which might mean that we miss most active requests in prometheus/grafana, but perhaps there is a slice of time of activity that might give us a hint about usage at any given time?